app/services/report_service.rb

```diff
@@ -20,7 +20,7 @@ def self.make_key(*values)
   end
 
   def self.make_source_key(url)
-    make_key(url.gsub(UrlRegex, '\1'))
+    url.gsub(UrlRegex, '\1')
```

Comment:
Does the cloud function replace the dots with underscores when it makes the top level source document name?  Or is this going to change the top level source document names?

Either approach seems fine, I'm just curious.

Reply:
based on other code it seems likely the keys now have dots.

Reply:
Sorry for the late answer, I didn't check this email, but I saw on Slack that this PR was merged.
Yes, I changed underscores to dots, so we can assume that the source is a host. That makes the code in the client-side app a bit simpler (no magic transformations required to get a source name from URL).